test/force-auto.py.in

```diff
@@ -14,23 +14,23 @@
 #
 # For each distribution, we test these factors:
 #
-#   - whether or not --force is given (2)
+#   - the value of --force (absent, fakeroot, seccomp) (3)
 #   - whether or not preparation for --force is already done (2)
 #   - commands that (4)
-#       - don't need --force, and fail
-#       - don't need --force, and succeed
+#       - don’t need --force, and fail
+#       - don’t need --force, and succeed
 #       - apparently need --force but in fact do not
 #       - really do need --force
 #
-# This would appear to yield 2×2×4 = 16 tests per distribution. However:
+# This would appear to yield 3×2×4 = 24 tests per distribution. However:
 #
-#   1. We only try pre-prepared images for "really need" commands with --force
-#      given, to save time, so it's at most 9 potential tests.
+#   1. We only try pre-prepared images for “really need” commands with --force
+#      given, to save time, so it’s at most 9 potential tests.
 #
-#   2. Some distros don't have any pre-preparation step, so that test doesn't
-#      make sense.
+#   2. The pre-preparation set doesn’t make sense for some distros or for
+#      --force=seccomp.
 #
-#   3. Some distros don't have an "apparently need" command determined.
+#   3. Some distros don’t have an “apparently need” command determined.
```

Comment:
This sounds awkward to me.  What about:  "`We've not yet determined an "apparently need" command for some distros.`?"  or "`The "apparently need" command for some distros has yet to be determined.`?"